package.json

```diff
@@ -30,7 +30,7 @@
     "babel-preset-es2015": "^6.9.0",
     "babel-preset-stage-1": "^6.5.0",
     "babel-register": "^6.9.0",
-    "babylon": "^7.0.0-beta.47",
+    "@babel/parser": "^7.0.0o",
```

Comment:
This seems like a weird version number. Typo?

Reply:
```suggestion
    "@babel/parser": "^7.0.0",
```